static/app/views/performance/traceDetails/utils.tsx

```diff
@@ -39,7 +39,12 @@ function getBaseTraceUrl(
       ? TRACE_SOURCE_TO_NON_INSIGHT_ROUTES[source]
       : 'traces';
 
-  return normalizeUrl(`/organizations/${organization.slug}/${url}`);
+  const routeSuffix =
```

Comment:
This was the only place that I didn't use the `makeTracesPathname` util since it is building the route more dynamically. Not sure if there is a better way to do this